pkg/controller/infinispan/service_site.go

```diff
@@ -49,16 +49,21 @@ func SiteService(siteServiceName string, infinispan *ispnv1.Infinispan, scheme *
 		}
 	}
 
-	siteService := corev1.Service{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      siteServiceName,
-			Namespace: infinispan.ObjectMeta.Namespace,
-			Annotations: map[string]string{
-				"service.beta.kubernetes.io/aws-load-balancer-backend-protocol": "tcp",
-			},
-			Labels: lsService,
+	objectMeta := metav1.ObjectMeta{
+		Name:      siteServiceName,
+		Namespace: infinispan.ObjectMeta.Namespace,
+		Annotations: map[string]string{
+			"service.beta.kubernetes.io/aws-load-balancer-backend-protocol": "tcp",
```

Comment:
This was hard-coded from before; I don't really know what it was needed for?
Does it need to be hard-coded anymore if `annotations` can be specified from the `CRD`?

Reply:
It specific to run xsite mode on AWS, not sure that we should move this annotation to the customer site as it's now possible as we should minimize the work which end user need to be done manually.
We can easily add this predefined annotation to the customer provided.

Reply:
> It specific to run xsite mode on AWS, not sure that we should move this annotation to the customer site as it's now possible as we should minimize the work which end user need to be done manually.
> We can easily add this predefined annotation to the customer provided.

Okay, i'll leave it here. If the user needs to override `annotations` then they will need to include this one as well.